.eslintrc

```diff
@@ -1,39 +1,9 @@
+---
 parser: "babel-eslint"
 
-env:
-
-    ###########################################################################
-    #                                                                         #
-    #   ENVIRONMENT: if you write code that will be executed in one of the    #
-    #   following environments, the value for that environment should be      #
-    #   set to true.                                                          #
-    #                                                                         #
-    ###########################################################################
-
-    browser: true
-    node: true
-    amd: false
-    mocha: true
-    jasmine: false
-
-globals:
-
-    ###########################################################################
-    #                                                                         #
-    #   GLOBALS: ESLint will assume the following variables are declared      #
-    #   globally; other variables will require explicit declaration.          #
-    #                                                                         #
-    ###########################################################################
-
-    require: true
```

Comment:
Obviated when you use env mocha. ESlint uses the globals package below.

https://github.com/sindresorhus/globals/blob/master/globals.json#L683-L703